src/containers/auth/SignedOutLinks.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./index.css";
+
+const SignedOutLinks = () => {
+  return (
+    <div className="link">
+      <Link to="/log-in">Sign In</Link>
```

Comment:
lets make these routes /login and /signup. usually we avoid hyphens in urls whenever possible (it doesnt look good). for example, look at FB, google, etc you'll never see a hyphen in the url unless its part of a search query or username